lua/metals.lua

```diff
@@ -156,7 +156,7 @@ M.info = function()
     end
     table.insert(output, "")
     table.insert(output, "## Helpful links")
-    table.insert(output, "  - https://gitter.im/scalameta/metals-vim")
+    table.insert(output, "  - https://discord.gg/FaVDrJegEh")
```

Comment:
Do you know if this link is one that expires after a certain amount of time, or is this one that was created under "expires never"? We want to make sure we're not including one that will expire.

Also, can we add another entry with the matrix bridge? So just add another line pointing to the matrix link.

https://matrix.to/#/#scalameta/vim-users:matrix.org

Reply:
Added matrix: https://matrix.to/#/#scalameta:vim-users